test/functional/shada/shada_spec.lua

```diff
@@ -264,4 +264,21 @@ describe('ShaDa support code', function()
     session:close()
     os.remove('NONE')
   end)
+
+  it('does not store unlisted buffer', function()
+    skip(is_os('win'))
+
+    local fname = funcs.getcwd() .. '/file'
+    meths.set_var('__fname', fname)
+    nvim_command('silent! edit `=__fname`')
```

Comment:
why silent with "!"? that silences errors...

Reply:
Removed `silent!`.
I've created this test by mostly coping another one. The other one did a silent write.